browser-extension/README.md

```diff
@@ -46,14 +46,24 @@ DB_PASSWORD: XXXXXXXXXXX
 
 ```
 git clone https://github.com/tattle-made/OGBV.git
-cd browser-extension/
+cd OGBV/browser-extension/
```

Comment:
@duggalsu  should we rename this to "Uli", given that is the new repo name?

Reply:
done!